vote-interface/src/state/vote_state_v4.rs

```diff
@@ -92,6 +92,20 @@ impl VoteStateV4 {
         }
     }
 
+    pub fn new(vote_init: &VoteInitV2, clock: &Clock) -> Self {
```

Comment:
I'm not sure how this is being used downstream already or how the transition is being planned, but was this breaking change intended?

Reply:
Ok great, thanks! Sorry for the noise then